website/docs/developers/device/encryption.md

```diff
@@ -1,16 +1,15 @@
 ---
-id: crypto
-title: Encryption in Meshtastic
-sidebar_label: Crypto
+id: encryption
+title: Meshtastic encryption
+sidebar_label: Encryption
 ---
```

Comment:
The protobufs will need to be updated as they contain a link to this page.

Reply:
PR done https://github.com/meshtastic/Meshtastic-protobufs/pull/20